packages/react-app/src/App.jsx

```diff
@@ -226,26 +240,16 @@ function App(props) {
 
       <Switch>
         <Route exact path="/">
-          {/* pass in any web3 props to this Home component. For example, yourLocalBalance */}
-          <Home
-            yourLocalBalance={yourLocalBalance}
-            writeContracts={writeContracts}
-            readContracts={readContracts}
-            address={address}
-            injectedProvider={injectedProvider}
-            userSigner={userSigner}
-          />
+          <Verify {...routeProps} />
         </Route>
-        <Route exact path="/polygons">
-          {/* pass in any web3 props to this Home component. For example, yourLocalBalance */}
-          <Polygons
-            yourLocalBalance={yourLocalBalance}
-            writeContracts={writeContracts}
-            readContracts={readContracts}
-            address={address}
-            injectedProvider={injectedProvider}
-            userSigner={userSigner}
-          />
+        <Route exact path="/create">
+          <CreateZone {...routeProps} />
+        </Route>
+        <Route exact path="/ethdenver">
+          <Home {...routeProps} />
+        </Route>
+        <Route exact path="/dune">
+          <Dune {...routeProps} />
         </Route>
         <Route path="/history">
           <Subgraph
```

Comment:
Seems like this file basically captures an overview of what you did.

You made a Dune route to see our analytics.
You changed Polygons to CreateZone.

Is that right?

Reply:
Yep! That's basically it.